src/lib/util/Util.ts

```diff
@@ -95,4 +99,17 @@ export class Util {
 	static pad(number: number): string {
 		return (number < 10 ? '0' : '') + number;
 	}
+
+	static async getResponse(value: string, interaction: { client: Client | GClient }) {
+		const languagePlugin = Plugins.get('@gcommands/plugin-language');
+
+		if (languagePlugin === null) return (interaction.client as GClient).responses[value];
```

Comment:
Duplicate `return (interaction.client as GClient).responses[value];`, not needed

Reply:
It's required because languagePlugin is null

Reply:
`if (languagePlugin !== null) {}`

Dont return in the if statement under false conditions, and one return at the end

Reply:
Okay 👍